OpenRA.Mods.Common/Widgets/SupportPowerTimerWidget.cs

```diff
@@ -49,7 +50,14 @@ public SupportPowerTimerWidget(World world)
 
 		public override void Tick()
 		{
-			texts = powers.Select(p =>
+			var displayedPowers = powers.Where(p =>
+			{
+				var owner = p.Instances[0].Self.Owner;
```

Comment:
https://github.com/OpenRA/OpenRA/pull/11177/files#diff-f65c5dc3568e124821d5e68818c9c218R64 it should be.